Makefile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2018 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: test
+test:
+	./check-fmt.sh
```

Comment:
can you move these scripts into a hack/ directory?

Reply:
For kaniko I pretty much copied over [hack](https://github.com/GoogleContainerTools/skaffold/tree/master/hack) from skaffold, it has a bunch of validation tests (boilerplate, dep, etc)